thorlcr/activities/indexread/thindexread.cpp

```diff
@@ -179,7 +182,8 @@ class CIndexReadBase : public CMasterActivity
                         rfn.getPath(remotePath);
                         unsigned crc = 0;
                         lastPart->getCrc(crc);
-                        keyIndex.setown(createKeyIndex(remotePath.str(), crc, false, 0));
+                        size32_t blockedIOSize = getEffectiveBlockSize(remotePath);
```

Comment:
I think this new function (getEffectiveBlockSize) can be replaced by call to the existing helper : findPlaneAttrFromPath